contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
The OSD never started (`osdmap e3: 1 osds: 0 up, 0 in`). Here is where it fails

```
[box36][WARNIN] INFO:ceph-disk:Running command: /usr/bin/ceph-osd --cluster ceph --mkfs --mkkey -i 0 --monmap /tmp/ceph/activate.monmap --osd-data /tmp/ceph --osd-journal /tmp/ceph/journal --osd-uuid 4ace486e-c5c7-4a88-8603-4f7aa958ab43 --keyring /tmp/ceph/keyring
[box36][WARNIN] 2015-05-18 15:10:12.749974 7fcd796df7c0 -1 memstore(/tmp/ceph) /tmp/ceph
[box36][WARNIN] 2015-05-18 15:10:12.753142 7fcd796df7c0 -1 created object store /tmp/ceph journal /tmp/ceph/journal for osd.0 fsid 65224421-549b-4c63-9db7-b44b02b0e8b2
[box36][WARNIN] 2015-05-18 15:10:12.753452 7fcd796df7c0 -1 auth: error reading file: /tmp/ceph/keyring: can't open /tmp/ceph/keyring: (2) No such file or directory
[box36][WARNIN] 2015-05-18 15:10:12.753700 7fcd796df7c0 -1 created new key in keyring /tmp/ceph/keyring
```

I'd run the script on your local box with the same parameters to verify that it works. I ran into similar weirdness on travis-ci which was often due to some restriction being placed on the environment, but i'm not immediately seeing the problem in this case.